src/ResourceManager/Profile/Commands.Profile/Context/ImportAzureRMContext.cs

```diff
@@ -14,7 +14,9 @@
 
 using Microsoft.Azure.Commands.Common.Authentication;
 using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Common.Authentication.Core;
```

Comment:
@markcowl I think this namespace needs to move down below into the `#ifdef` for .NET Standard

Reply:
fixed